### DISCUSSION TITLE
Online tracking on iOS

### DISCUSSION THREAD

**Discussion by Jeremy-VVu** [osmandapp/OsmAnd#13850] [Maps Related]
I love this app, it provides a solution for rendering instant paths online, but what bothers me is, Why can't use "online tracking" on iOS mobile phone?
![173167](https://user-images.githubusercontent.com/59771521/156719039-f8eaac7a-89a1-4259-bf38-f5e3be9b7c2b.jpg)

**Comment by vshcherb** [comment]
It's not implemented yet

**Reply by Jeremy-VVu** [reply]
Thanks for your reply and look forward to deploying this feature soon.

**Comment by clementmas** [comment]
Is the development of the Online Tracking feature planned for iOS? Where can we follow / participate?

Or is this not even considered?
